src/System.Private.ServiceModel/src/System.Private.ServiceModel.csproj

```diff
@@ -22,6 +22,11 @@
   <PropertyGroup Condition="'$(IsNETNativeLibrary)' != 'true'">
     <DefineConstants>$(DefineConstants);FEATURE_CORECLR</DefineConstants>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(TargetsWindows)' == 'true'">
+    <DefineConstants>$(DefineConstants);TARGETS_WINDOWS</DefineConstants>
+	<ProjectJson>$(MsBuildThisFileDirectory)windows\project.json</ProjectJson>
+	<ProjectLockJson>$(MsBuildThisFileDirectory)windows\project.lock.json</ProjectLockJson>
+  </PropertyGroup>
```

Comment:
I wouldn't add a condition. I think I should just add the other one in the same item group. The None group simply says to show the file in the VS solution explorer. It has no functional effect. I'll add it in so it doesn't get forgotten.